src/llvm/pass_builder_options.cr

```diff
@@ -0,0 +1,28 @@
+{% skip_file if LibLLVM::IS_LT_130 %}
+
+class LLVM::PassBuilderOptions
+  def initialize
+    @options = LibLLVM.create_pass_builder_options
```

Comment:
💭 There are some options that can be configured here, but I didn't bother to experiment with this: https://llvm.org/doxygen/c_2Transforms_2PassBuilder_8h.html